cvat-ui/src/components/create-cloud-storage-page/cloud-storage-form.tsx

```diff
@@ -419,64 +414,45 @@ export default function CreateCloudStorageForm(props: Props): JSX.Element {
         if (providerType === ProviderType.GOOGLE_CLOUD_STORAGE && credentialsType === CredentialsType.KEY_FILE_PATH) {
             return (
                 <Form.Item
+                    name='key_file'
                     {...internalCommonProps}
                     label={(
-                        <Tooltip title='You can specify path to key file or upload key file.
-                                If you leave these fields blank, the environment variable will be used.'
+                        <Tooltip title='You can upload a key file.
+                                If you leave this field blank, the environment variable
+                                GOOGLE_APPLICATION_CREDENTIALS will be used.'
                         >
                             Key file
+                            <Button
+                                href='https://cloud.google.com/docs/authentication/getting-started#setting_the_environment_variable'
+                                target='_blank'
+                                type='link'
+                                className='cvat-cloud-storage-help-button'
+                            >
+                                <QuestionCircleOutlined />
+                            </Button>
                         </Tooltip>
 
                     )}
                 >
                     <Space align='start' className='cvat-cloud-storage-form-item-key-file'>
-                        <Form.Item
-                            name='key_file_path'
-                            noStyle
+                        <Dragger
+                            accept='.json, application/json'
+                            multiple={false}
+                            maxCount={1}
+                            fileList={
+                                uploadedKeyFile ? [{ uid: '1', name: 'key.json' }] : []
```

Comment:
Why do you show unreal file name?

Reply:
Because in this case we control output and avoid a situation like this:
![image](https://user-images.githubusercontent.com/49038720/144219816-a0cc1250-1d18-4282-a164-f87d65224401.png)

Reply:
What about this one? 
![image](https://user-images.githubusercontent.com/40690378/144227573-f3bd2fc6-0cad-4c15-8285-89780390d105.png)